masq_lib/Cargo.toml

```diff
@@ -12,9 +12,11 @@ actix = "0.7.9"
 clap = "2.33.3"
 const_format = "0.2.22"
 crossbeam-channel = "0.5.1"
+csv = "1.3.0"
```

Comment:
I think csv shouldn't be added to the `masq_lib` production tree, at least if I understand things correctly.

It sounds like it's only used in a test. If it absolutely has to be in this `Cargo.toml`, then, perhaps it should be in the test tree only.